Px.Utils/Validation/ContentValidation/ContentValidator.UtilityMethods.cs

```diff
@@ -40,31 +40,26 @@ private static Dictionary<KeyValuePair<string, string>, string[]> FindDimensionV
                         e.Key.FirstSpecifier.Equals(dimension, StringComparison.Ordinal));
                     if (valuesEntry is not null)
                     {
-                        string[] values = valuesEntry.Value.Split(syntaxConf.Symbols.Value.ListSeparator);
-                        for(int i = 0; i < values.Length; i++)
-                        {
-                            values[i] = SyntaxValidationUtilityMethods.CleanString(values[i], syntaxConf);
-                        }
+                        List<string> values = SyntaxValidationUtilityMethods.GetListItemsFromString(
+                            valuesEntry.Value, 
+                            syntaxConf.Symbols.Value.ListSeparator, 
+                            syntaxConf.Symbols.Value.StringDelimeter);
+                        values = CleanListOfStrings(values, syntaxConf);
 
                         variableValues.Add(
                             new KeyValuePair<string, string> ( language, dimension ),
-                            values
+                            [..values]
                             );
                     }
                     else
                     {
-                        ValidationFeedback feedback = new (
-                                ValidationFeedbackLevel.Error,
-                                ValidationFeedbackRule.VariableValuesMissing,
-                                0,
-                                0,
-                                $"{dimension}, {language}"
-                                );
+                        KeyValuePair<ValidationFeedbackKey, ValidationFeedbackValue> feedback = new(
+                            new(ValidationFeedbackLevel.Error,
+                                ValidationFeedbackRule.VariableValuesMissing),
+                            new(filename, 0, 0, $"{dimension}, {language}")
```

Comment:
Maybe make the coordinates nullable, so it is easier to distinguish when the error actually is at position 0 and when there is no meaningful position.